crates/libtiny_tui/src/key_map.rs

```diff
@@ -254,8 +254,8 @@ impl<'de> Deserialize<'de> for MappedKey {
                             ch => Key::Ctrl(single_key(ch)?),
                         },
                         "shift" => match k2 {
-                            "up" => Key::ShiftUp,
-                            "down" => Key::ShiftDown,
+                            "up" => Key::Shift(Arrow::Up),
+                            "down" => Key::Shift(Arrow::Down),
```

Comment:
I think this should be updated with left/right cases?

Reply:
Yes, sorry. Moved to draft to add more keys (such as Ctrl/Shift/Alt + F Keys).